docs/gettingstarted/index.rst

```diff
@@ -159,6 +159,9 @@ Getting started with Flocker
          .. version-code-block:: console
 
             you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile && \
+              curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/cluster.crt && \
```

Comment:
What I mean is that the curl command should fail if the server responds with 404.
It'll cause confusion if the certificate and key files and up full of junk.
But create a follow up issue for that.